uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-balance-tab.component.js

```diff
@@ -37,8 +37,11 @@ define([
      * and watches for events coming from child components.
      */
     (function init () {
-      loadDependencies();
-      watchForEvents();
+      initWatchers();
+
+      loadDependencies().then(function () {
+        vm.loading.component = false;
```

Comment:
I wonder if `vm.loading.component = false;` can be moved to `loadDependencies()`?